fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -5265,6 +5272,8 @@ class FileBackupAgentImpl {
 	//   lockDB: if set lock the database with randomUid before performing restore;
 	//           otherwise, check database is locked with the randomUid
 	//   incrementalBackupOnly: only perform incremental backup
```

Comment:
Though this is an existing issue, while you're here let's fix this comment.

```suggestion
	//   incrementalBackupOnly: only perform incremental restore, by only applying mutation logs
```